AdminApi/src/AdminApi/Program.cs

```diff
@@ -79,7 +80,8 @@ static WebApplication CreateApp(string[] args)
     var app = builder.Build();
     Configure(app);
 
-    app.MigrateDbContext<AdminApiDbContext>();
+    if (app.Configuration.GetValue<bool>("RunMigrations"))
+        app.MigrateDbContext<AdminApiDbContext>();
```

Comment:
shouldn't all migrations be in the migrator?

Reply:
This is only done if the "RunMigrations" config flag is set. It's for local development. This way you don't need to run the database migrator.
It's only a short term solution. Long term we will add the database migrator. I just didn't find a quick solution, so I postponed it.

Reply:
gotcha. can you make sure that this is only possible in development?

Reply:
Hmm, I could. But I don't think it's worth it. Because the users won't have permission to create/update/drop tables in production environments anyway.